examples/api/hello_world.py

```diff
@@ -20,5 +20,17 @@ def forward(self, x):
 pytorch_model = SmallModel(input_size, output_size)
 inputs = {"x": torch.rand(input_size)}
 
-benchit(pytorch_model, inputs, device="nvidia")
-benchit(pytorch_model, inputs, device="x86")
+print ("Benchmarking local x86 CPU...")
```

Comment:
The API already has a print like this here: https://github.com/groq/mlagility/blob/78efdb182877ccf52798ff2b90f180e64aaf983f/src/mlagility/api/api.py#L169. Converge them? 

It should probably say: "Benchmarking {build_name} on {device_type} at {ip}" or similar